xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -85,32 +84,8 @@ void CGUIDialogPVRTimerSettings::OnSettingChanged(const CSetting *setting)
   const std::string &settingId = setting->GetId();
   if (settingId == SETTING_TMR_ACTIVE)
     m_bTimerActive = static_cast<const CSettingBool*>(setting)->GetValue();
-  if (settingId == SETTING_TMR_RADIO || settingId == SETTING_TMR_CHNAME_TV || settingId == SETTING_TMR_CHNAME_RADIO)
-  {
-    if (settingId == SETTING_TMR_RADIO)
-    {
-      tag->m_bIsRadio = static_cast<const CSettingBool*>(setting)->GetValue();
-      m_selectedChannelEntry = 0;
-    }
-    else
-      m_selectedChannelEntry = static_cast<const CSettingInt*>(setting)->GetValue();
-
-    std::map<std::pair<bool, int>, int>::iterator itc = m_channelEntries.find(std::make_pair(tag->m_bIsRadio, m_selectedChannelEntry));
-    if (itc != m_channelEntries.end())
-    {
-      CPVRChannelPtr channel =  g_PVRChannelGroups->GetChannelById(itc->second);
-      if (channel)
-      {
-        tag->m_iClientChannelUid = channel->UniqueID();
-        tag->m_iClientId         = channel->ClientID();
-        tag->m_bIsRadio          = channel->IsRadio();
-        tag->m_iChannelNumber    = channel->ChannelNumber();
-       
-        // Update channel pointer from above values
-        tag->UpdateChannel();
-      }
-    }
-  }
+  if (settingId == SETTING_TMR_CHNAME_TV || settingId == SETTING_TMR_CHNAME_RADIO)
```

Comment:
No, but I changed as minimal as possible as I knew that @ksooo was rewriting it.
Should I drop to 1 settingId then?